astropy/table/jsviewer.py

```diff
@@ -59,24 +59,26 @@ class Conf(_config.ConfigNamespace):
 
 IPYNB_JS_SCRIPT = """
 <script>
-%(sorting_script1)s
-require.config({{paths: {{
-    datatables: '{datatables_url}'
-}}}});
-require(["datatables"], function(){{
-    console.log("$('#{tid}').dataTable()");
-    %(sorting_script2)s
-    $('#{tid}').dataTable({{
+{sorting_script1}
```

Comment:
Yeah, I think `ruff`'s f-string reformatter maybe a bit too aggressive @taldcroft are these changes okay?

Reply:
Okay... well looking at this more closely I think it converted to a `.format` string, to the best of my eye it looks okay. I think it is doubling `{{` to escape the `.format` `{}` recognition ...

Honestly, I think the formatting via the `%` in this case is a bit easier to read.

Reply:
Yeah, I don't think ruff should be touching this.

Reply:
They should be equivalent. But I am a bit confused why it doesn't go all the way to a f-string.

Reply:
I'd rather this file not be touched. I will just close this PR because whoever following up on this will have to squash out the changes anyway. Thanks!